samples/snippets/quickstart.py

```diff
@@ -0,0 +1,266 @@
+            You can check the list of available public images using:
+            $ gcloud compute images list
+        network_name: Name of the network you want the new instance to use.
+            For example: global/networks/default - if you want to use the
+            default network.
+
+    Returns:
+        Instance object.
+    """
+    instance_client = compute_v1.InstancesClient()
+
+    # Every machine requires at least one persistent disk
+    disk = compute_v1.AttachedDisk()
+    initialize_params = compute_v1.AttachedDiskInitializeParams()
+    initialize_params.source_image = (
+        source_image  # "projects/debian-cloud/global/images/family/debian-10"
+    )
+    initialize_params.disk_size_gb = "10"
+    disk.initialize_params = initialize_params
+    disk.auto_delete = True
+    disk.boot = True
+    disk.type_ = compute_v1.AttachedDisk.Type.PERSISTENT
+
+    # Every machine needs to be connected to a VPC network.
+    # The 'default' network is created automatically in every project.
+    network_interface = compute_v1.NetworkInterface()
+    network_interface.name = network_name
+
+    # Collecting all the information into the Instance object
+    instance = compute_v1.Instance()
+    instance.name = instance_name
+    instance.disks = [disk]
+    full_machine_type_name = f"zones/{zone}/machineTypes/{machine_type}"
+    instance.machine_type = full_machine_type_name
+    instance.network_interfaces = [network_interface]
+
+    # Preparing the InsertInstanceRequest
+    request = compute_v1.InsertInstanceRequest()
+    request.zone = zone
+    request.project = project_id
+    request.instance_resource = instance
+
+    print(f"Creating the {instance_name} instance in {zone}...")
+    operation = instance_client.insert(request=request)
+    # wait_result = operation_client.wait(operation=operation.name, zone=zone, project=project)
+    operation = wait_for_operation(operation, project_id)
+    if operation.error:
+        pass
+    if operation.warnings:
+        pass
+    print(f"Instance {instance_name} created.")
+    return instance
+
+
+# [END compute_instances_create]
+
+
+# [START compute_instances_delete]
+def delete_instance(project_id: str, zone: str, machine_name: str) -> None:
+    """
+    Sends a delete request to GCP and waits for it to complete.
+
+    Args:
+        project_id: ID or number of the project you want to use.
+        zone: Name of the zone you want to use, for example: us-west3-b
+        machine_name: Name of the machine you want to delete.
+    """
+    instance_client = compute_v1.InstancesClient()
+
+    print(f"Deleting {machine_name} from {zone}...")
+    operation = instance_client.delete(
+        project=project_id, zone=zone, instance=machine_name
+    )
+    operation = wait_for_operation(operation, project_id)
+    if operation.error:
+        pass
+    if operation.warnings:
+        pass
+    print(f"Instance {machine_name} deleted.")
```

Comment:
Can the sample show error handling? (Even if it is just printing out the error/warnings?).

Reply:
Done.